components/03-modules/footer/_footer.scss

```diff
@@ -0,0 +1,172 @@
+$footer__bg-color                    : $gray-darker !default;
+$footer__bg-color--screen-l          : $gray-dark !default;
+$footer__font-color                  : $white !default;
+$footer__transition                  : $transition-base !default;
+
+$footer__newsletter-max-width        : 400px !default;
+$footer__newsletter-width--screen-l  : 30% !default;
+
+$footer__links-width--screen-l       : 70% !default;
+$footer__link-color-after-hover      : $color-primary !default;
+$footer__link-height-after-hover     : 8px !default;
+$footer__link-font-weight-after-hover: $font-weight-bold !default;
+
+$footer__bottom-bar-background       : $gray-darker !default;
+$footer__bottom-bar-color            : $gray !default;
+$footer__bottom-bar-font-size        : $font-size-small !default;
+
+
+.footer {
+    background-color: $footer__bg-color;
+    padding: $spacer--medium 0;
+
+    @include mq($screen-m) {
+        padding-bottom: 0;
+    }
+
+    @include mq($screen-l) {
+        background-color: $footer__bg-color--screen-l;
+    }
+
+    &__handler {
+        margin: 0 auto $spacer--medium;
+
+        @include mq($screen-m) {
+            margin: 0 auto;
+        }
+
+        @include mq($screen-l) {
+            display: flex;
+            flex-flow: row wrap;
+        }
+    }
+
+    &__newsletter {
+        margin: 0 auto $spacer--medium;
+        max-width: $footer__newsletter-max-width;
+
+        @include mq($screen-l) {
+            width: $footer__newsletter-width--screen-l;
+            order: 1;
+            padding-top: $spacer--medium;
+        }
+    }
+
+    &__links {
+        @include mq($screen-l) {
+            width: $footer__links-width--screen-l;
+        }
+
+        .dropdown-list {
+            @include mq($screen-m) {
+                padding: 0;
+            }
+        }
+
+        .dropdown-list__item {
+            @include mq($screen-m) {
+                border-bottom: 0;
+            }
+        }
+    }
+
+    &__links-list {
+        padding: $spacer--medium 0;
+
+        .link {
+            display: inline-block;
+            position: relative;
+            overflow: hidden;
+            z-index: 1;
+            color: $footer__font-color;
+            text-decoration: none;
+            transition: $footer__transition;
+
+            &:after {
+                content: '';
+                position: absolute;
+                bottom: 1px;
+                left: 0%;
+                height: $footer__link-height-after-hover;
+                background-color: $footer__link-color-after-hover;
+                width: 100%;
+                z-index: -1;
+                transform: scaleX(0);
```

Comment:
Is there a real need for this ? I see that you scale it at focus and hover, but do we really need to use it here ?

Reply:
It's link variant but on other PR if it will be merged I will use this code here but for now, leave this whole code for link class

Reply:
if you say so